server/server.go

```diff
@@ -560,34 +560,34 @@ func (s *Server) CodeToken(creds oidc.ClientCredentials, sessionKey string) (*jo
 	return jwt, refreshToken, nil
 }
 
-func (s *Server) RefreshToken(creds oidc.ClientCredentials, scopes scope.Scopes, token string) (*jose.JWT, error) {
+func (s *Server) RefreshToken(creds oidc.ClientCredentials, scopes scope.Scopes, token string) (*jose.JWT, string, error) {
```

Comment:
Raw claims are encoded immediately [here](https://github.com/coreos/dex/blob/0e94e7625551a0e2c455de6d2568489e02e00f98/server/http.go#L534). But the interface [OIDCServer](https://github.com/coreos/dex/blob/0e94e7625551a0e2c455de6d2568489e02e00f98/server/server.go#L46) uses jose.JWT in more places. Do you prefer to change it?